helm-charts/helm-api/values.yaml

```diff
@@ -103,3 +102,19 @@ volumes:
   mongoClientCertSecretName: icp-mongodb-client-cert
   rudderCertsName: rudder-certs
   rudderCertsSecretName: rudder-secret
+tiller:
+  defaultAdminUser: admin
+  image:
+    name: tiller
+    pullPolicy: IfNotPresent
+    repository:  quay.io/opencloudio/tiller
+    tag: v2.16.1-icp-3.2.2
+  nodePort: 31514
+  resources:
```

Comment:
I don't see this variable being used anywhere? The tiller deployment manifest has hard coded resources

Reply:
those are default values for chart ..they are exposed in deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml
can be configured/overlay with OperandConfig

Reply:
right, but does the CPU and memory requests/limits serve any purpose in the values.yaml file?

Reply:
the resource parameters serve the same purpose as other parameters, that they are configurable. This is similar to other helm operators where resource parameters are exposed as configurable options. However, I am not sure the importance of the presence in values.yaml. For consistency sake, I'm adding to the values.yaml in addition to the cr.yaml

Reply:
oh my bad, I thought the resources variable in values.yaml wasn't being used. But I see it being used now